### PR TITLE
XWIKI-15653: Prevent users from deleting/moving/renaming pages containing XClass

### PR DESCRIPTION
**Pull request by surli**
* Fix checkstyle issue

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ private void answer(HttpServletRequest request, Job job, List<String> jobId, Obj
         try {
             this.beans.populate(question, extractParameters(request, question));
         } catch (PropertyException e) {
-            throw new ResourceReferenceHandlerException("Failed to populate question object for job with id " + jobId, e);
+            throw new ResourceReferenceHandlerException("Failed to populate question object for job with id " + jobId);
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java]: this is not good, you've removed the information from the nested exception! :)

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java]: BTW our best practice is to surround with `[...]`, see https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HLoggingBestPractices

Comment (surli) [reply xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java]: Actually I added the information from the nested exception in #885. So I just restore the old behaviour. But if you think it's needed for the future I'll put it back.

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java]: Yes we need it, we always need to forward the nested exception. Thanks!

Comment (surli) [reply xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java]: I used a String.format then, tell me if it's ok.

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-job/xwiki-platform-job-handler/src/main/java/org/xwiki/job/handler/internal/question/QuestionJobResourceReferenceHandler.java]: That's perfect (I didn't dare tell you to use a String.format ;)).

**Pull request merged**